### DISCUSSION TITLE
NPCBots Source and Development

### DISCUSSION THREAD

**Discussion by cstrickland98** [trickerer/Trinity-Bots#410] [General]
Is the source code for just the patch accessible? I wanted to see how everything is put together so I can attempt to port it to a later version of TC.

**Comment by trickerer** [comment]
Source code for both base repositories is public. It is strange that you didn't notice them right away since they are listed along with this one.

https://github.com/trickerer?tab=repositories
https://github.com/trickerer/TrinityCore-3.3.5-with-NPCBots
https://github.com/trickerer/AzerothCore-wotlk-with-NPCBots

Note that base branches of base repositories mirrored there are outdated, I don't push those normally.

**Reply by cstrickland98** [reply]
> Ah yes. I saw those, but I meant more of just the changes that you introduced with NPCBots to make it easier to see what was added vs what is already there in one of those cores. Do you have a suggestion on a good approach to do something like that? 

I figured out what I need actually. Thank you for responding to a very dumb questions lol
